contracts/swaps/connectors/SwapsImplKyber.sol

```diff
@@ -107,6 +107,12 @@ contract SwapsImplKyber is State, ISwapsImpl {
         }
     }
 
+    function revokeApprovals(address[] memory tokens) public {
```

Comment:
`SwapsImplKyber` is called with BZXProxy context using delegate call, e.g in `setSupportedTokens` `setSwapApprovals` is called. the same should be true for `revokeApprovals` if we wanna handle this situation. 
`setSupportedTokens` should be callable by guardians but thats another disscusion

Reply:
we can add a revokeApprovals on ProtocolSettings.sol to address this